packaging/makedef.py

```diff
@@ -87,7 +87,7 @@ def removeprefix(self: str, prefix: str, /) -> str:
     else:
         print(f"{libname.stem.replace('.', '_')} {{")
         print("\tglobal:")
-        print(";\n".join(sorted(set([f'\t\t{i}' for i in dump]))))
+        print(";\n".join(sorted(set([f'\t\t{i}' for i in dump]))) + ';')
```

Comment:
Looking at this, would it not make sense to add a `\n` on the end of this added semicolon so that `local:` goes on a new line? LGTM otherwise.

Reply:
Ammendment: turns out `print()` sticks a new line on the end anyway, making the review note entirely pointless.. marking resolved.